module/common/pylgettext.py

```diff
@@ -58,5 +58,10 @@ def find(domain, localedir=None, languages=None, all=False):
         return results
 
 #Is there a smarter/cleaner pythonic way for this?
-translation.func_globals['find'] = find
-origfind.func_globals['find'] = origfind
+try:
+    translation.func_globals['find'] = find
+    origfind.func_globals['find'] = origfind
+except AttributeError:
+    translation.__globals__['find'] = find
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['translation' may be undefined, or defined from star imports: gettext](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=4846282)